themes/vald/layouts/_default/index.html

```diff
@@ -3,6 +3,11 @@
     <div class="top__content">
       <p class="top__title">Vald</p>
       <p class="top__lead">A Highly Scalable Distributed Vector Search Engine</p>
+      <div class="top__linklist">
+        <a href="https://vald.vdaas.org/docs/tutorial/get-started/" class="top__linkitem top__linkitem--getstarted"><span class="top_linktext">Get Started</span></a>
+        <a href="https://vald.vdaas.org/docs/" class="top__linkitem top__linkitem--docs"><span class="top__linktext">Docs</span></a>
```

Comment:
```suggestion
        <a href="https://vald.vdaas.org/docs/" class="top__linkitem top__linkitem--docs"><span class="top__linktext">Documents</span></a>
```
こっちでもいいかも？
文言の揺らぎなくすならこのままでもOKです。